### PR TITLE
Proofreading and fixing minor typos of documentations

### PR DESCRIPTION
**Pull request by CodeWithEmad**
Hi Regis.
I always read the tutor’s documentation and every time, I saw some small issues in the grammar, it bothers me, cause the tutor is such a great tool and it should have the highest standards. so, on a weekend, I checked all of the documentation for proofreading and grammar issues using my knowledge and Grammarly.

these changes happened in this PR:
```
lots of "the" and commas were added.
the wrong usage of "by" changed to "with".
some words were spelled wrong. 

```
Lots of words changed to the original form. like:
```
url -> URL
kubernetes -> Kubernetes
http -> HTTP
https -> HTTPS
html -> HTML
nginx -> Nginx
memecache -> Memecache
mongodb -> MongoDB
pypi -> PyPI
redis -> Redis
rabbitmq -> RabbitMQ
```

I noticed that you use "s" instead of "z" in some words, like customise and initialise. Personally, I don't think it's wrong but the Grammarly was using them with the letter "z". checked it with google and it said:

A lot of words follow the spelling rule where British English uses s, while American English uses z. Use customise for your British English audience, and use customize for American English and Canadian English readers.

Because the word "initialise" was used in some parts of the code like [jobs.py](https://github.com/overhangio/tutor/blob/master/tutor/jobs.py) or [compose.py](https://github.com/overhangio/tutor/blob/master/tutor/commands/compose.py), I didn't change them. on the other hand, both customise and customize were used in the documentation. I want you to decide either to change one to the other or leave them alone?

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ With Tutor, all Open edX deployment parameters are stored in a single ``config.y
 
     cat "$(tutor config printroot)/config.yml"
 
-By default, this file contains only the required configuration parameters for running the platform. Optional configuration parameters may also be specified to modify the default behaviour. To do so, you can edit the ``config.yml`` file manually::
+By default, this file contains only the required configuration parameters for running the platform. Optional configuration parameters may also be specified to modify the default behavior. To do so, you can edit the ``config.yml`` file manually::
```
Comment (regisb) [inline docs/configuration.rst:22]: Could we check the vocab and grammar with UK English? This is the convention that I try to adopt across Tutor. For instance, "behaviour" is British while "behavior" is US: https://www.wordreference.com/enfr/behaviour

File: docs/faq.rst [hunk 2]
```diff
@@ -6,7 +6,7 @@ FAQ
 What is Tutor?
 --------------
 
-Tutor is an open source distribution of `Open edX <https://open.edx.org>`_. It uses the original code from the various Open edX repositories, such as `edx-platform <https://github.com/openedx/edx-platform/>`_, `cs_comments_service <https://github.com/openedx/cs_comments_service>`_, etc. and packages everything in a way that makes it very easy to install, administer and upgrade Open edX. In particular, all services are run inside Docker containers.
+Tutor is an open-source distribution of `Open edX <https://open.edx.org>`_. It uses the original code from the various Open edX repositories, such as `edx-platform <https://github.com/openedx/edx-platform/>`_, `cs_comments_service <https://github.com/openedx/cs_comments_service>`_, etc. and packages everything in a way that makes it very easy to install, administer and upgrade Open edX. In particular, all services are run inside Docker containers.
```
Comment (regisb) [inline docs/faq.rst:9]: TIL "open-source" is the adjective, "open source" is the noun https://en.wikipedia.org/wiki/Open_source

File: docs/index.rst [hunk 3]
```diff
@@ -57,7 +57,7 @@ License
 
 This work is licensed under the terms of the `GNU Affero General Public License (AGPL) <https://github.com/overhangio/tutor/blob/master/LICENSE.txt>`_.
 
-The AGPL license covers the Tutor code, including the Dockerfiles, but not the content of the Docker images which can be downloaded from https://hub.docker.com. Software other than Tutor provided with the docker images retain their original license.
+The AGPL license covers the Tutor code, including the Dockerfiles, but not the content of the Docker images which can be downloaded from https://hub.docker.com. Software other than Tutor provided with the docker images retains their original license.
```
Comment (regisb) [inline docs/index.rst:60]: In this case "software" is plural so it should be "retain".

File: docs/k8s.rst [hunk 4]
```diff
@@ -5,7 +5,7 @@ Kubernetes deployment
 
 With the same docker images we created for :ref:`single server deployment <local>` and :ref:`local development <development>`, we can launch an Open edX platform on Kubernetes. Always in 1 click, of course :)
 
-A word of warning: managing a Kubernetes platform is a fairly advanced endeavour. In this documentation, we assume familiarity with Kubernetes. Running an Open edX platform with Tutor on a single server or in a Kubernetes cluster are two very different things. The local Open edX install was designed such that users with no prior experience with system administration could still launch an Open edX platform. It is *not* the case for the installation method outlined here.
+A word of warning: managing a Kubernetes platform is a fairly advanced endeavor. In this documentation, we assume familiarity with Kubernetes. Running an Open edX platform with Tutor on a single server or in a Kubernetes cluster are two very different things. The local Open edX install was designed such that users with no prior experience with system administration could still launch an Open edX platform. It is *not* the case for the installation method outlined here.
```
Comment (regisb) [inline docs/k8s.rst:8]: Same comment here: preserve "endeavour" vs "endeavor".

File: docs/k8s.rst [hunk 5]
```diff
@@ -34,22 +34,22 @@ By default, Tutor deploys a `LoadBalancer <https://kubernetes.io/docs/concepts/s
 
     tutor k8s start caddy
 
-Get the external IP of this services::
+Get the external IP of these services::
```
Comment (regisb) [inline docs/k8s.rst:37]: "this service"

File: docs/k8s.rst [hunk 6]
```diff
@@ -67,11 +67,11 @@ On Minikube, the dashboard is already installed. To access the dashboard, run::
...
 
 Basically, the whole platform is described in manifest files stored in ``$(tutor config printroot)/env/k8s``. There is also a ``kustomization.yml`` file at the project root for `declarative application management <https://kubectl.docs.kubernetes.io/guides/config_management/introduction/#declarative-application-management>`__. This allows us to start and update resources with commands similar to ``kubectl apply -k $(tutor config printroot) --selector=...`` (see the ``kubectl apply`` `official documentation <https://kubectl.docs.kubernetes.io/references/kubectl/apply/>`__).
 
-The other benefit of ``kubectl apply`` is that it allows you to customise the Kubernetes resources as much as you want. For instance, the default Tutor configuration can be extended by a ``kustomization.yml`` file stored in ``$(tutor config printroot)/env-custom/`` and which would start with::
+The other benefit of ``kubectl apply`` is that it allows you to customize the Kubernetes resources as much as you want. For instance, the default Tutor configuration can be extended by a ``kustomization.yml`` file stored in ``$(tutor config printroot)/env-custom/`` and which would start with::
```
Comment (regisb) [inline docs/k8s.rst:74]: Again: keep "customise", "initialise", and more generally UK English words.

File: docs/plugins/api.rst [hunk 7]
```diff
@@ -16,7 +16,7 @@ config
 
 The ``config`` attribute is used to modify existing and add new configuration parameters:
 
-* ``config["add"]`` are key/values that should be added to the user-specific ``config.yml`` configuration. Add there passwords, secret keys and other values that do not have a default value.
+* ``config["add"]`` are key/values that should be added to the user-specific ``config.yml`` configuration. Add their passwords, secret keys, and other values that do not have a default value.
```
Comment (regisb) [inline docs/plugins/api.rst:19]: It's really "there".

File: docs/plugins/api.rst [hunk 8]
```diff
@@ -47,7 +47,7 @@ This configuration from the "myplugin" plugin will set the following values:
 patches
 ~~~~~~~
 
-Plugin patches affect the rendered environment templates. In many places the Tutor templates include calls to ``{{ patch("patchname") }}``. This grants plugin developers the possibility to modify the content of rendered templates. Plugins can add content in these places by adding values to the ``patches`` attribute.
+Plugin patches affect the rendered environment templates. In many places the Tutor templates include calls to ``{{ patch("patchname") }}``. These grants plugin developers the possibility to modify the content of rendered templates. Plugins can add content in these places by adding values to the ``patches`` attribute.
```
Comment (regisb) [inline docs/plugins/api.rst:50]: "This"

**Review by regisb — CHANGES_REQUESTED** [review]
As much as it pains me to have a bot hightlight all my English mistakes, I'm very grateful for this PR which will considerably improve the docs. Thanks a lot! Please check my various comments. In particular, we should prefer UK over US English.

**Comment by CodeWithEmad** [comment]
WOW. lot's of comments. I'll change them today.

File: CHANGELOG.md [hunk 9]
```diff
@@ -83,7 +83,7 @@ Note: Breaking changes between versions are indicated by "💥".
   - Add a ``app.kubernetes.io/component: loadbalancer`` label to the LoadBalancer service.
   - Add ``app.kubernetes.io/name`` labels to all services.
   - Preserve the LoadBalancer service in ``tutor k8s stop`` commands.
-  - Wait for the caddy deployment to be ready before running initialisation jobs.
+  - Wait for the caddy deployment to be ready before running                     jobs.
```
[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply CHANGELOG.md:86]: Oops. No, Just a mistake.

File: CHANGELOG.md [hunk 10]
```diff
@@ -151,7 +151,7 @@ Note: Breaking changes between versions are indicated by "💥".
 - [Bugfix] Fix segmentation fault during `tutor config save` on Mac OS M1 (#473). Thanks @ghassanmas!
 - [Bugfix] Fix a bug that prevented connecting to external MongoDB instances.
 - [Improvement] Make sure that the logo included in email notifications (including discussion responses) is the same as the site logo.
-- [Bugfix] Install IPython directly from pypi instead of installing it from source (the reason it was installed from source is no longer relevant). The effect of this shall speed up the process of building the openedx-dev Docker image.
+- [Bugfix] Install IPython directly from PyPI instead of installing it from the source (the reason it was installed from the source is no longer relevant). The effect of this shall speed up the process of building the openedx-dev Docker image.
```
[the remark this answers is not in the document]
Comment (CodeWithEmad) [reply CHANGELOG.md:154]: No argument there.

File: docs/configuration.rst (hunk 1 above)
Comment (CodeWithEmad) [reply docs/configuration.rst:22]: ok, I wasn't aware of the UK over US situation. behaviour It is! :)

**Review by regisb — APPROVED** [review]

**Comment by regisb** [comment]
Thanks for the quick response @CodeWithEmad!

**Pull request merged**